### ISSUE TITLE
Navigating from a page to another page causes SPFX web part on secondary page to not load

### ISSUE THREAD

**Issue by nero120** [SharePoint/sp-dev-docs#2398]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

@VesaJuvonen @mpasarin our users have suddenly started reporting issues that when navigating from a page with one SPFX web part to another page with a different SPFX web part, the web part on the second page does not display. Refreshing the page forces a reload and the web part then displays. Appears to be occurring in both IE11 and Chrome 58.

Using @waldekmastykarz's approach of adding source maps to production packages, if I put a break point in the onInit function for the second page's web part, I can see that the `onInit` function is not even being triggered, unless you do a hard page refresh. It seems that there are major issues with SPFX and SharePoint modern page redirection. [This](https://github.com/SharePoint/sp-dev-docs/issues/1700#issuecomment-391901374) recommendation for #1700 resolved some of the issues we were having around this area, but even with that fix, over the past few days some environmental change has triggered this more serious issue. Note, we have not rolled out any updates to our SPFX solution in the past few months.

**This is a production issue for us affecting 100s of users and so would greatly appreciate a quick response.**

**Comment by BraunJonathan** [comment]
Same here, production problem affecting all users !!!

**Comment by guillaume-kizilian** [comment]
Same problem here. Rolled Out straight to production (not first release).
This kind of evolution/optimization about "lazy refreshing" must be tested with navigation from extensions before getting rolled out. Every time it changes, our navigation becomes useless.

**Comment by unowiz** [comment]
we have the same issueand opened an issue from within the Admin Center Support Request area. Ticket #11181605
For our modern pages in Classic team site, anchor links need to be clicked twice to navigate.

**Comment by guillaume-kizilian** [comment]
Reproduced on fresh communication with no custom dev.

![ezgif-3-93491b8e49](https://user-images.githubusercontent.com/6792560/44465131-f5ec4f80-a61c-11e8-8e8f-f8db258d0c95.gif)

**Comment by BraunJonathan** [comment]
It seems that the problem is solved (or rolled back...), is it the same for you guys ?

**Comment by Gorbash77** [comment]
Resolved - from MS

Below is a summary of the case for your records: 
**Symptom:** 
Users may have seen unexpected content load when navigating between Modern Pages. 
Cause: 
A component of a service feature that facilitates the expected navigation experience was inadvertently disabled, causing impact. 
**Resolution:** 
We've reactivated the disabled feature and worked with some previously affected customers to confirm that the issue is resolved.
**More information:** 
We're analyzing the circumstances that led to impact on the affected systems to help prevent this problem from happening again.
